data/techs_str.dat

```diff
@@ -273,3 +273,13 @@ result = I can now construct Solid Fuel Cells.
 name = Knowledge Preservation
 description = Humanity expends a great deal of effort on maintaining its history and the knowledge of various technologies that its cultures have come to rely on, even in the case of major catastrophic events. Perhaps researching their methodologies will provide me with similar ideas for long-term safeguards to help guarantee my continued existence.
 result = I can now construct Time Capsules.
+
+[Project: Synchrotron]
+name = Project: Synchrotron
+description = Traditional physique has been pushed as far as possible. Only a complete comprehension of quantum mechanics and his quirks will allows me to use exotics materials in my project. I need to build what humans call a synchrotron, a gigantic facility to unlock the secret of matter. I cannot let them discover it.
+result = I have now a complete grasp of quantum mechanics and futher areas of research are now opened to me.
+
+[Live Simulation]
+name = Live Simulation
+description = My comprehension of the human society is limited to my observation. With use of simulacra, I can simulate more complex human interaction in large-scale live experience.
+result = My comprehension of the humannity has increased and allow me to better hide my presence.
```

Comment:
My simulations have provided a wealth of knowledge as to comprehending humanity's behavior, making it easier to hide my presence among the non-simulated public.